src/components/dashboard/data/utils.js

```diff
@@ -45,3 +45,10 @@ export const getStatusMetadata = ({
     date: endDateStr,
   };
 };
+
+export const setLocalStorageData = (localStorageKeyValue) => {
+  const [key, value] = localStorageKeyValue;
+  global.localStorage.setItem(key, value);
+};
+
+export const hasLocalStorageData = (localStorageKey) => !!global.localStorage.getItem(localStorageKey);
```

Comment:
[nit/suggestion] I might recommend calling `global.localStorage.setItem` and `global.localStorage.getItem` directly, versus calling them from within these util functions. The function signature would be slightly more evident / readable if the key and value were separate function args, but then it'd largely just be a wrapper around something that's already a single line. Any particular reason we need these util functions around `global.localStorage`?

Reply:
Removed. I can see why a single line helper would be redundant 👍🏽